### PR TITLE
v9.2 Changelog

### PR DESCRIPTION
**Pull request by amyblais**
To be added: list of contributors; potential changes to the list of known issues.

### CODE REVIEW

File: source/install/self-managed-changelog.md [hunk 1]
```diff
@@ -6,11 +6,73 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
...
 - [Release v9.0 - Major Release](#release-v9-0-major-release)
 - [Release v8.1 - Extended Support Release](#release-v8-1-extended-support-release)
 - [Release v7.8 - Extended Support Release](#release-v7-8-extended-support-release)
 
+## Release v9.2 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: November 16, 2023**
+
+### Important Upgrade Notes
+ - Fixed Data Retention Policies to run jobs when there is any "Custom retention policy" enabled even when the "Global retention policy" is set to "keep-forever". Before this fix, the Enabled Custom Policies wouldn't run as long as Global Policy was set to "keep-forever" or was disabled. After the fix, the custom policies will run automatically even when the Global Policy is set to "keep-forever". Once the server is upgraded, posts may unintentionally be deleted. Admins should make sure to disable all custom policies before upgrading, and then re-enable them again after upgrading.
```
Comment (amyblais) [inline source/install/self-managed-changelog.md:20]: @vish9812 Is there any special testing that needs to be done to test this breaking change, or do the test steps in the Jira ticket cover everything https://mattermost.atlassian.net/browse/MM-44002?

Comment (vish9812) [reply source/install/self-managed-changelog.md:20]: @amyblais Yes, those steps are pretty much it.

File: source/install/self-managed-changelog.md [hunk 2]
```diff
@@ -6,11 +6,78 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
...
+### Compatibility
+ - Updated minimum required Edge version to 116+.
+
+### Improvements
+
+#### User Interface (UI)
+ - Improved readability by displaying system messages on multiple lines when editing a channel header.
+ - Combined "joined/left" event types in system messages.
+ - Added a new user preference to disable webapp prefetching via **Settings > Advanced > Allow Mattermost to prefetch channel posts**.
+ - Pre-packaged Focalboard plugin version [v7.11.4](https://github.com/mattermost/focalboard/releases/tag/v7.11.4).
```
Comment (esethna) [inline source/install/self-managed-changelog.md:33]: I would leave this out given the deprecation. It was only a patch release

**Review by agnivade — APPROVED** [review]

**Review by isacikgoz — APPROVED** [review]

**Review by wiersgallak — APPROVED** [review]

**Review by stylianosrigas — APPROVED** [review]

**Review by marianunez — APPROVED** [review]

**Review by sbishel — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

**Review by vish9812 — APPROVED** [review]

File: source/install/self-managed-changelog.md [hunk 3]
```diff
@@ -6,11 +6,77 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
...
+
+### config.json
+ - Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to all plans:
+ - Under ``LogSettings`` in ``config.json``:
+   - Added a new configuration setting ``MaxFieldSize`` to add the ability to size-limit log fields during logging.
+
+### API Changes
+ - Added ``origin_client to the mattermost_api_time`` metrics.
```
Comment (ThiefMaster) [inline source/install/self-managed-changelog.md:62]:
```suggestion
 - Added ``origin_client`` to the ``mattermost_api_time`` metrics.
```

image

**Comment by amyblais** [comment]
Changelog has been added via https://github.com/mattermost/docs/pull/6632.

**Pull request merged**